src/libraries/SharesMathLib.sol

```diff
@@ -14,8 +14,9 @@ library SharesMathLib {
 
     /// @dev The number of virtual shares has been chosen low enough to prevent overflows, and high enough to ensure
     /// high precision computations.
-    /// @dev Warning: The assets to which virtual borrow shares are entitled behave like unrealizable bad debt, but it
-    /// is assumed the borrow share price stays low enough not to inflate these assets to a significant value.
+    /// @dev Virtual shares can never be redeemed for the assets they are entitled to, but it is assumed the share price
+    /// stays low enough not to inflate these assets to a significant value.
+    /// @dev Warning: The assets to which virtual borrow shares are entitled behave like unrealizable bad debt.
```

Comment:
same here, missing `to`

Reply:
```suggestion
    /// @dev Warning: The assets to which virtual borrow shares are entitled to behave like unrealizable bad debt.
```

Reply:
The "to" is in "to which"

Reply:
And we argued with @MerlinEgalite this formulation was simpler that the former one: https://github.com/morpho-org/morpho-blue/pull/639#discussion_r1419245370

Reply:
ahh my bad, got it :)

Reply:
My 2 cents: it could be written in a more clean way? The concept you want to express/address are:

1) virtual shares (supply and borrow) cannot be redeemed (withdrawn/repaid) because they are not owned by anyone
2) virtual supply shares "steal" interest from suppliers
3) virtual borrow shares are indeed unrealizable bad debt
4) point 2 and 3 does not impact the code because "it is assumed the share price stays low enough not to inflate these assets to a significant value."

Reply:
I agree that's another clear way to put it
But I also think the current way is clear enough / is just fine